src/libraries/Microsoft.PowerFx.Interpreter/EvalVisitor.cs

```diff
@@ -341,6 +341,35 @@ public override FormulaValue Visit(RecordFieldAccessNode node, SymbolContext con
                 return left;
             }
 
+            if (left is CustomObjectValue cov)
+            {
+                if (cov.Impl.IsObject)
+                {
+                    if (cov.Impl.TryGetProperty(node.Field.Value, out var res))
+                    {
+                        return new CustomObjectValue(node.IRContext, res);
+                    }
+                    else
+                    {
+                        return new ErrorValue(node.IRContext, new ExpressionError()
```

Comment:
We should return BlankValue for missing keys in an object, so we're consistent with First(Table({a: 3), {b: 4})) => Blank()

Reply:
Done